syntax = "proto3";

package ts_messaging;

enum  CalcType {
    PRIVATEKEYSHARE = 0 ;
    LITTLEK         = 1 ;
    ALPHA           = 2 ;
  } 